permamodel/components/bmi_frost_number.py

```diff
@@ -229,6 +233,29 @@ def update(self):
         self._model.calculate_frost_numbers()
         self._values['frostnumber__air'] = self._model.air_frost_number
 
+    def update_frac(self, time_fraction):
```

Comment:
Here's what I was thinking:
  - the time in the model is a floating point (referring to year) [Note: might change this to days...]
  - the model timestep is a single, full year
  - a fractional timestep would be part of a year
  - the model has no problem with it's current time being a non-integer year...
    - ...but the model won't update its values until a full year has passed, whether as a result of a full time step or a series of fractional timesteps adding up to >= a full year
  - the frost number could, of course, be redefined to be calculated in a different way, say from the preciding year's data or something, but that's not how it's coded now